crates/spirv-std/src/ray_tracing.rs

```diff
@@ -288,16 +288,30 @@ impl RayQuery {
     }
 
     /// Terminates further execution of a ray query; further calls to
-    /// [`Self::proceed`] will return `false`.The value returned by any prior
+    /// [`Self::proceed`] will return `false`. The value returned by any prior
     /// execution of [`Self::proceed`] with the same ray query object must have
-    /// been true. Refer to the client API specification for more details.
```

Comment:
I removed this like just because it wasn't consistent with the rest of the docs and seems to have just been copied from https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/KHR/SPV_KHR_ray_query.asciidoc.